### PR TITLE
fix: mf-791 view address on explorer

### PR DESCRIPTION
**Pull request by UncleBill**

### CODE REVIEW

File: packages/mask/src/components/InjectedComponents/ProfileCard/ProfileBar.tsx [hunk 1]
```diff
@@ -113,6 +113,7 @@ export const ProfileBar = memo<ProfileBarProps>(
         const [walletMenuOpen, setWalletMenuOpen] = useState(false)
 
         const formattedAddress = Others?.formatAddress(address ?? '', 4) ?? ''
+        const addressLink = address && Others?.explorerResolver.addressLink(ChainId.Mainnet, address)
```
Comment (guanbinrui) [inline packages/mask/src/components/InjectedComponents/ProfileCard/ProfileBar.tsx]: Does it safe to dead code Mainnet?

Comment (UncleBill) [reply packages/mask/src/components/InjectedComponents/ProfileCard/ProfileBar.tsx]: Now use current chain id.

File: packages/mask/src/components/InjectedComponents/ProfileCard/ProfileBar.tsx [hunk 2]
```diff
@@ -127,7 +129,7 @@ export const ProfileBar = memo<ProfileBarProps>(
                         <Link
```
Comment (guanbinrui) [inline packages/mask/src/components/InjectedComponents/ProfileCard/ProfileBar.tsx]: The `Link` will ref to an unexpected URL if `addressLink` is absent.

Comment (UncleBill) [reply packages/mask/src/components/InjectedComponents/ProfileCard/ProfileBar.tsx]: Updated. Now check it before using.

**Review by guanbinrui — APPROVED** [review]

**Pull request merged**